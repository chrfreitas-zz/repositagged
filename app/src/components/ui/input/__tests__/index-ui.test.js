import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Input from '../index';

describe('Input view', () => {
  it('should match with snapshot', () => {
    const renderer = new ShallowRenderer();
    const result = renderer.render(
      <Input placeholder="Digite aqui seu nome" />
    );
    expect(result).toMatchSnapshot();
  });
});
