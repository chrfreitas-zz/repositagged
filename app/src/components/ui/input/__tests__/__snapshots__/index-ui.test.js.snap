// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input view should match with snapshot 1`] = `
<input
  className="input"
  placeholder="Digite aqui seu nome"
  type="text"
/>
`;
